### ISSUE TITLE
Setup issue

### ISSUE THREAD

**Issue by bregnery** [CMSDAS/b2g-long-exercise#5]
The shell script being accessed with curl fails due to a public key error. It fails for me on both lxplus and lpc.

```
$ bash <(curl -s https://raw.githubusercontent.com/lcorcodilos/CombineHarvester/master/CombineTools/scripts/sparse-checkout-ssh.sh)
+ set -e
+ '[' -z x ']'
+ pushd /afs/cern.ch/user/b/bregnery/public/das2020b2g/CMSSW_10_6_14/src
~/public/das2020b2g/CMSSW_10_6_14/src ~/public/das2020b2g/CMSSW_10_6_14/src
+ mkdir CombineHarvester
+ cd CombineHarvester
+ git init
Initialized empty Git repository in /afs/cern.ch/user/b/bregnery/public/das2020b2g/CMSSW_10_6_14/src/CombineHarvester/.git/
+ git remote add origin git@github.com:lcorcodilos/CombineHarvester.git
+ git config core.sparsecheckout true
+ echo CombineTools/
+ git pull origin master
Warning: Permanently added the RSA host key for IP address '140.82.121.3' to the list of known hosts.
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

**Comment by bregnery** [comment]
This can be fixed by requiring students to generate public keys and add them to their gitHub accounts. Do we want to do that? If so, we should include some instructions.

**Comment by bregnery** [comment]
Fixed in #6
